src/components/Popup/Popup.js

```diff
@@ -18,7 +18,7 @@ const Popup = () => {
 
     if (layer.get('popupComponent')) {
       if (typeof layer.hidePopup === 'function') {
-        return features.find(f => !layer.hidePopup(f));
+        return features.find(f => !layer.hidePopup(f, layer, featureInfo));
```

Comment:
Is there already the layer in featureInfo object ?

Reply:
'featureInformation' holds potentially several 'info', and 'layer' comes from the specifically tested 'info'.